_rules/table-headers-attribute-refer-to-data-cells.md

```diff
@@ -0,0 +1,208 @@
+---
+name: `headers` attribute only refers to `cells` in the same `table` element
+
+test_type: atomic
+
+description: |
+  This rule checks that the `headers` attribute must only refer to `cells` in the same `table` element
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to `td` and `th` elements with a `headers` attribute and within a `table` element, that is either [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+### Expectation
+
+Each target element with the `headers` attribute refers to other `cells` of the same `table` element.
+
+## Assumptions
+
+- This test assumes that `headers` pointing to an `idref` of a non-existent element is a failure.
```

Comment:
Is this really needed? If the id does not exist then it is not in the same table, right?